feature: Add link to ingestion API reference docs PUL-625 (#65)
codacy/pulse-user-docs: docs/cli/pushing-data-to-pulse.md

-   [**Make sure that you have an API key**](https://app.pulse.codacy.com/integrations/cli){: target="_blank"} provided by Pulse to identify your organization and authorize you to send data to Pulse.

> -   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are reporting events from providers that only support webhooks. For these situations, you can [use the Pulse Ingestion API instead](https://ingestion.pulse.codacy.com/v1/api-docs){: target="_blank"}.

Reviewer: ```suggestion
-   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when you have the data inside an application. For these situations, you can [use the Pulse Ingestion API instead](https://ingestion.pulse.codacy.com/v1/api-docs){: target="_blank"}.
```

I think the webhooks part is not worth mentioning since it is provided automatically inside the integrations.
And you need to get the proper url from our app and cannot do the setup without it.

Author: You're right, and I also like that the explanation becomes simpler. :+1:
